framework/contracts/native/version-control/src/contract.rs

```diff
@@ -30,28 +33,13 @@ pub struct VcResponse;
 pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> VCResult {
     let to_version: Version = CONTRACT_VERSION.parse()?;
 
-    let vc_addr_raw = deps.storage.get(b"fac");
-    if let Some(vc_addr) = vc_addr_raw {
-        let vc_addr: Option<cosmwasm_std::Addr> = cosmwasm_std::from_json(vc_addr)?;
-
-        CONFIG.update(deps.storage, |mut cfg| {
-            // Save factory address to a new place
-            cfg.account_factory_address = vc_addr;
-            // Check if fee requires in migration
-            if cfg
-                .namespace_registration_fee
-                .as_ref()
-                .map(|fee| fee.amount.is_zero())
-                .unwrap_or(false)
-            {
-                // registration_fee is Option now, but was 0 in previous version
-                cfg.namespace_registration_fee = None;
-            }
-            VCResult::Ok(cfg)
-        })?;
-    }
-    // Remove old factory
-    deps.storage.remove(b"fac");
+    let old_config = Item::<OldConfig>::new(from_utf8(CONFIG.as_slice())?).load(deps.storage)?;
```

Comment:
What's this slice utf logic? @Kayanski

Reply:
I wanted to make sure the config names were the same to avoid errors when copying the storage key

Reply:
there's `deps.storage.get` if you want to use same key.